lib/kontena/plugin/cloud/node/create_command.rb

```diff
@@ -19,9 +19,10 @@ class Kontena::Plugin::Cloud::Node::CreateCommand < Kontena::Command
   option "--type", "TYPE", "Node type", required: true
   option "--region", "REGION", "Region (us-east-1, eu-west-1, defaults to current platform region)"
   option "--ssh-key", "SSH_KEY", "Path to ssh public key"
+  option "--platform", "PLATFORM", "Platform name", environment_variable: 'KONTENA_PLATFORM'
```

Comment:
I would not expose this as an option here. Maybe just check `KONTENA_PLATFORM` env in `default_platform` ?

Reply:
I have refactored implementation as you proposed